pkgs/os-specific/linux/s6-linux-utils/default.nix

```diff
@@ -28,16 +23,10 @@ in stdenv.mkDerivation rec {
   ];
 
   postInstall = ''
-    mkdir -p $doc/share/doc/s6-networking/
-    mv doc $doc/share/doc/s6-networking/html
-  '';
+    # remove all s6 executables from build directory
+    rm $(find -name "s6-*" -type f -mindepth 1 -maxdepth 1 -executable)
```

Comment:
Why is cleaning the build directory necessary?

Reply:
It’s an experiment of mine, being more strict with how build artifacts are handled. The idea is that every file has to be either moved to an output or deleted.
If a future update introduces e.g. an `example` folder, the maintainer will be alerted that there’s new files that should probably go to an output.

Reply:
Do you think it is a common problem? Did you found any examples where this was the case for Skaware's software?

Reply:
I don’t know if it’s a common problem, other builds don’t register when new files are added. I strongly suspect it, since many (many!) packages don’t copy documentation, READMEs, or even manpages to their outputs.

Reply:
I am not so sure if many people are reading documentation from the nix store except for man/info pages though.